tests/macsec/test_macsec.py

```diff
@@ -205,3 +229,109 @@ def check_mka_establishment():
         disable_macsec_port(duthost, port_name)
         disable_macsec_port(nbr["host"], nbr["port"])
         delete_macsec_profile(nbr["host"], profile_name)
+
+
+class TestInteropProtocol():
+    '''
+    Macsec interop with other protocols
+    '''
+
+    def test_port_channel(self, duthost, ctrl_links):
+        '''Verify lacp
+        '''
+        ctrl_port, _ = ctrl_links.items()[0]
+        pc = find_portchannel_from_member(ctrl_port, get_portchannel(duthost))
+        assert pc["status"] == "Up"
+
+        # Remove ethernet interface <ctrl_port> from PortChannel interface <pc>
+        duthost.command("sudo config portchannel member del {} {}".format(
+            pc["name"], ctrl_port))
+        assert wait_until(6, 1, 0, lambda: get_portchannel(
+            duthost)[pc["name"]]["status"] == "Dw")
+
+        # Add ethernet interface <ctrl_port> back to PortChannel interface <pc>
+        duthost.command("sudo config portchannel member add {} {}".format(
+            pc["name"], ctrl_port))
+        assert wait_until(6, 1, 0, lambda: find_portchannel_from_member(
+            ctrl_port, get_portchannel(duthost))["status"] == "Up")
+
+    def test_lldp(self, duthost, ctrl_links, profile_name):
+        '''Verify lldp
+        '''
+        LLDP_ADVERTISEMENT_INTERVAL = 30  # default interval in seconds
+        LLDP_HOLD_MULTIPLIER = 4  # default multiplier number
+        LLDP_TIMEOUT = LLDP_ADVERTISEMENT_INTERVAL * LLDP_HOLD_MULTIPLIER
+
+        # select one macsec link
+        for ctrl_port, nbr in ctrl_links.items():
+            # TODO: vsonic vm has issue on lldp
+            if not isinstance(nbr["host"], EosHost):
+                pytest.skip("test_lldp has issue with vsonic neighbor")
```

Comment:
Please add 
`| jq '.DEVICE_METADATA.localhost.mac |= sub("(?<=^\w)[26aeAE]"; "c")'`
after 
https://github.com/Azure/sonic-mgmt/blob/3cc50dcc7d25ebc4ee33baafaf3f009bc32aa175/ansible/roles/sonic/tasks/vsonic.yml#L27
, I believe that could fix this issue in vsonic
![image](https://user-images.githubusercontent.com/18609639/158001144-dc402157-da35-4ee5-b248-3972db43095f.png)